### PR TITLE
Sample goal/question/metric

### PR DESCRIPTION
**Pull request by rrrutledge**
Using the templates at https://github.com/InnerSourceCommons/managing-inner-source-projects/pull/21.

@jeffabailey to take a look.

### CODE REVIEW

File: measuring/questions/who-uses.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Who uses the InnerSource project?
+
+Depending on the InnerSource project, usage could look something like:
```
Comment (jeffabailey) [inline measuring/questions/who-uses.md:3]: What does usage refer to?

I presume it means usage of the GQM, and this describes how you would measure it?

File: measuring/metrics/usage-count.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# Usage count
```
Comment (jeffabailey) [inline measuring/metrics/usage-count.md:1]: May make it easier to understand what you're looking at when navigating the graph.

```suggestion
# **Metric:** Usage count
```

File: measuring/questions/who-uses.md (hunk 1 above)
Comment (rrrutledge) [reply measuring/questions/who-uses.md:3]: I'm glad you asked.  This references to usage of _any_ InnerSource project.  If someone has a goal to reduce duplication of work then one question that supports that goal is how widely used are their InnerSource projects.  Let me know if that makes sense?

Comment (jeffabailey) [inline measuring/questions/who-uses.md:3]:
```suggestion
Depending on the InnerSource project, usage of the project could look something like:
```

File: measuring/metrics/usage-count.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+# **Metric:** Usage count
+
+**Synopsis**: Count how many times the InnerSource project is used.
+Gain additional insight by gathering metadata on these usages such as business unit or timeframe.
+With this additional metadata we can see how lopsided or even the usage is spread across the company and across time.
+
+**Scale/UoM**: Ordinal (I think 😆).
```
Comment (jeffabailey) [inline measuring/metrics/usage-count.md:7]: Pending the outcome of the other PR, we might change this to **Data Type**:

**Review by jeffabailey — CHANGES_REQUESTED** [review]

File: measuring/questions/who-uses.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
+# Who uses the InnerSource project?
```
Comment (rrrutledge) [inline measuring/questions/who-uses.md:1]: Is "project" too large?  "Component" may be too large.

Comment (rrrutledge) [reply measuring/questions/who-uses.md:1]: There is some hierarchy summary - "Reuse pyramid".  @gruetter may know more, here.

File: measuring/metrics/usage-count.md (hunk 3 above)
Comment (jeffabailey) [reply measuring/metrics/usage-count.md:7]: We decided on *Unit of Measurement*.

File: measuring/questions/who-uses.md (hunk 4 above)
Comment (rrrutledge) [reply measuring/questions/who-uses.md:1]: We can't think of anything better.

**Pull request merged**